client/app/lib/util/proxifier.coffee

```diff
@@ -35,11 +35,7 @@ module.exports = class Proxifier
     subdomain = if isInProduction then 'p' else 'dev-p'
 
     # create the base url
-    # Since proxy is a service from koding.com we need to use it
-    # for on-premise version as well. otherwise all the on-premises
-    # users needs to run proxy server on their own ~GG
-    baseURL = "#{protocol}//#{subdomain}.koding.com/-"
-    # baseURL = "#{protocol}//#{subdomain}.#{globals.config.domains.base}/-"
+    baseURL = "#{protocol}//#{subdomain}.#{globals.config.domains.base}/-"
```

Comment:
I've split user proxy and tunnel proxy locations into separate groups. User proxy locations are in the web server group now. This is a much easier change to read in the `workers` module.

I've tested current changes in default and development environments.